woocommerce/woocommerce-gateway-stripe
# Add missing currency notice

Resolves #2006 

## Changes proposed in this Pull Request:

Adds a notice in the payment method list, in the settings, to indicate that the payment method requires a currency. The pill is only shown when the store currency differs from the payment method supported currencies.

As #2006 suggests, I also included support to WCPay multi-currency. I also raised an internal discussion (p1636395471041300-slack-CHG7MTCAF) to see if that's really what we want, once we currently don't support multi-currency alongside with Stripe on checkout.

![image](https://user-images.githubusercontent.com/10233985/140816474-a4d12bde-0c31-49d0-928b-1587a2fc7a79.png)








## Testing instructions

1. Ensure you have `_wcstripe_feature_upe_settings` option set to `yes`.
2. Set your store currency to `USD`, if it's not already.
3. Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
4. You **should see** a notice identical to the screen shot above.
5. Set your store currency to `EUR`.
6. Go back to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
7. You **should not see** any notices about the currency.





---


**Post merge**
